src/java.scripting/share/classes/module-info.java

```diff
@@ -26,10 +26,9 @@
 /**
  * Defines the Scripting API.
  *
- * <p> The JDK implementation of this module includes a language-independent
- * command-line script shell, <em>{@index jrunscript jrunscript tool}</em>,
- * that supports executing JavaScript and other languages if its corresponding
- * script engine is installed.
+ * <p> The <em>{@index jrunscript jrunscript tool}</em> included in the JDK
+ * implementation of this module, has been deprecated for removal since
+ * JDK 24 and will be removed in a future release.
```

Comment:
"included" is past sentence. I think it would be better to keep the original text and just add new a sentence to say that the tool has been deprecated for removal in a future release.

Reply:
Done, I've now updated the PR to retain the old text, plus add a new line noting about the deprecation.